README.md

```diff
@@ -4,7 +4,7 @@ Explore and open files in the current project.
 
 Press `cmd-\` to open/close the Tree view and `ctrl-0` to focus it.
 
-When the Tree view has focus you can press `a`, `m`, or `delete` to add, move
+When the Tree view has focus you can press `a`, `Shift-a`, `m`, or `delete` to add, move
```

Comment:
The convention in our documentation is for keyboard shortcuts to be in all lower case: `Shift-a` &rarr; `shift-a`